### DISCUSSION TITLE
Go to file in a new tab

### DISCUSSION THREAD

**Discussion by proivk** [pwntester/octo.nvim#360] [Q&A]
Is it possible to "go to file" in a new tab?
gf works perfectly on PR overview page, for instance. But would be nice to have option to open the file in a new tab (similar to "C-W gf" command)

**Comment by pwntester** [comment]
That should be a vim command and Octo should not interfere on it. Does `C-W gf` work for you when Octo is not installed?

**Reply by proivk** [reply]
It works even when Octo is installed. But it doesn't work in Octo PR overview window.
C-W gf results to "E446: No file name under cursor", gf works well though.

**Reply by pwntester** [reply]
Just tried and it works for me. Can you try with a minimal config?

**Reply by proivk** [reply]
Alright, I tried this on a very minimal config (only Octo plugin and its dependencies) and still it's not working.
The interesting thing is that C-Wgf works well in other Octo views (for exsmple Octo Review pane). The issue appears only on PR Overview window on the lines with code blocks.
